tests/codegen/try_question_mark_nop.rs

```diff
@@ -9,14 +9,11 @@
 use std::ops::ControlFlow::{self, Break, Continue};
 use std::ptr::NonNull;
 
-// FIXME: The `trunc` and `select` instructions can be eliminated.
 // CHECK-LABEL: @option_nop_match_32
 #[no_mangle]
 pub fn option_nop_match_32(x: Option<u32>) -> Option<u32> {
     // CHECK: start:
-    // CHECK-NEXT: [[TRUNC:%.*]] = trunc nuw i32 %0 to i1
-    // CHECK-NEXT: [[FIRST:%.*]] = select i1 [[TRUNC]], i32 %0
-    // CHECK-NEXT: insertvalue { i32, i32 } poison, i32 [[FIRST]]
+    // CHECK-NEXT: insertvalue { i32, i32 } poison, i32 %0, 0
```

Comment:
this is going to fail on LLVM less than 20

maybe it would be best to use revisions: https://rustc-dev-guide.rust-lang.org/tests/compiletest.html#revisions

Reply:
Shoot. I had meant to do just `CHECK:` on this line, which would work on both, but be a looser check. Would that be good enough? Or should I see if I can combine revisions with `min-llvm-version: 20` and `ignore-llvm-versions: 20-99`?

Reply:
Personally, I would give the revisions a spin, because otherwise this isn't validating the `trunc`/`select` are gone.

Reply:
Yeah, that looks correct! Thanks!